recipes/xnnpack/all/conanfile.py

```diff
@@ -63,7 +63,8 @@ def layout(self):
     def requirements(self):
         self.requires("cpuinfo/cci.20220228")
         self.requires("fp16/cci.20210320")
-        self.requires("pthreadpool/cci.20210218")
+        self.requires("pthreadpool/cci.20210218", transitive_headers=True)
```

Comment:
are you sure?

Reply:
Indeed, it's public: https://github.com/google/XNNPACK/blob/ed5f9c0562e016a08b274a4579de5ef500fec134/include/xnnpack.h#L15